src/helpers/add-pr-late-label.ts

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2022 Expedia, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { LATE_REVIEW } from '../constants';
+import { HelperInputs } from '../types/generated';
+import { octokit } from '../octokit';
+import { RestEndpointMethodTypes } from '@octokit/rest';
+import { each } from "bluebird";
+
+type PullResponseType = RestEndpointMethodTypes["pulls"]["get"]["response"];
+type ListResponseType = RestEndpointMethodTypes["pulls"]["list"]["response"];
+
+export class AddPrLateReviewLabels extends HelperInputs {
+  owner = '';
+  repo = '';
+}
+
+export const addPrLateReviewLabels = async ({ owner, repo }: AddPrLateReviewLabels) => {
+  var page = 1;
+  var pull_requests = await getPRs(owner, repo, page) as ListResponseType;
+
+  while (pull_requests.data.length > 0 ) {
+    if (pull_requests.status != 200) {
+      //loghere
+      continue;
+    }
+
+    const pr_data = pull_requests.data || [];
+  
+    // Loop through all of the issue numbers
+    await each(pr_data, async pull_request =>
+      await labelPullRequest( 
+        pull_request,
+        owner,
+        repo
+      )
+    );
+    // Setup next PR page
+    page += 1;
+    // Get all pull requests  
+    pull_requests = await getPRs(owner, repo, page);
+  }
+};
+
+const getPRs = async ( owner: string, repo: string, page: number) => {
+  return await octokit.pulls.list({
+    owner: owner,
+    repo: repo,
+    state: "open",
+    sort: "created",
+    per_page: 100,
+    page: page
+  }) as ListResponseType;
+}
+
+const labelPullRequest = async ( pull_request: any, owner: string, repo: string) => {
+  const pr = parseInt(pull_request.id);
+  // Get the PR
+  const pull_request_data = await octokit.pulls.get({
```

Comment:
We should already have the necessary PR data from our initial `pulls.list` call

Reply:
I wasnt able to find mergeable as a parameter returned by the list 
https://docs.github.com/en/rest/pulls/pulls#list-pull-requests
, it _does_ have the updated_at field however, so if we can find a way to get the mergeable, or find an alternative way to determine status, we can do away with the call on ilne 69

Reply:
Looks like we can actually use the `requested_reviewers` and `requested_teams` fields on each listed PR to determine if there are open requests out for a review. So we shouldn't need to make this additional call